loadouts/faction-list.sqf

```diff
@@ -141,14 +141,10 @@ switch (toUpper(_faction)) do {
         #include "factions\frwinter.sqf"
     };
 
-    case "FRWOOD": {
+    case "FRANCE": {
         #include "factions\frwood.sqf"
     };
 
-    case "FRDESERT": {
```

Comment:
Why is "FRWINTER" not removed?

Reply:
Different uniform type using TRYK mod, vanilla vest and RHS helmet, unless there is a way to completely change the loadout using the camo function it has to be in a different file.